Purchases.xcodeproj/project.pbxproj

```diff
@@ -1055,21 +1050,20 @@
 		37E3529AD4F30D0B6D56853B /* ProtectedExtensions */ = {
 			isa = PBXGroup;
 			children = (
-				37E35D29AD7BE9B81DBF6907 /* RCPromotionalOffer+Protected.h */,
+				3589D15524C21DBD00A65CBB /* RCAttributionTypeFactory+Protected.h */,
```

Comment:
is this only alphabetization? or are there files added / removed aside from RCEntitlementInfo headers?

Reply:
Alphabetization. It's an old habit of mine... everything gets alphabetized.

Reply:
I love having stuff alphabetized, was just asking because it's hard to figure out what the actual diff is, though.